packages/components/src/autocomplete/autocompleter-ui.js

```diff
@@ -63,6 +70,7 @@ export function getAutoCompleterUI( autocompleter ) {
 					id={ listBoxId }
 					role="listbox"
 					className="components-autocomplete__results"
+					ref={ popoverRef }
```

Comment:
Shouldn't this `ref` be applied to the `Popover` component, instead of this `div` ? Currently, clicking on the "padding" of the popover closes the autocomplete list

Reply:
Sounds good to me. I think the ref is already forwarded in `Popover`, so that should work fine.